### DISCUSSION TITLE
pyMOR Community Meeting 2023-04-12

### DISCUSSION THREAD

**Discussion by sdrave** [pymor/pymor#2010] [Announcements]
The next pyMOR Community Meeting will take place Wednesday, April 12, at 14:00 CEST and 8:00 EDT, online at:

    link: https://bbb.mpi-magdeburg.mpg.de/meeting/pymor
    password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [x] release process
- [x] pyMOR School 2023 organization
- [x] support email request "solder joints"
- [x] Python 3.10

**Comment by artpelling** [comment]
Minutes:

- release process
  - postponed discussion
- pyMOR School 2023 organization
  - funding is an issue although we do not have many expenses
  - check legal requirements for demanding a registration fee
  - if nothing works, we can ask for donations
  - tentative date: Sep 18-22 (check again and settle on it next time)
- support email
  - no news
- Python 3.10
  - we already support 3.10 but do not advertise it
  - we should update the badge #2011
- AOB
  - @sdrave works on simplifying the docker workflow
  - @pmli works on pre-commit hooks for Markdown checking
 
Next meeting: 2023-04-26, 12:00 UTC (14:00 CEST, 8:00 EDT)
Next moderator: @sdrave
